### ISSUE TITLE
256-mode not working on MacOS Terminal

### ISSUE THREAD

**Issue by dimays** [urwid/urwid#419]
##### Description:
This is what `palette_test.py` example displays on my Mac Os Terminal for 256 color mode:

![Screen Shot 2020-07-06 at 5 27 33 PM](https://user-images.githubusercontent.com/31248790/86657672-09ed9000-bfae-11ea-8376-2735e3dc5699.png)

And this is what the High Color Modes example from [the Tutorials page](http://urwid.org/tutorial/index.html) displays:

![Screen Shot 2020-07-06 at 5 30 43 PM](https://user-images.githubusercontent.com/31248790/86658176-75376200-bfae-11ea-868e-fe8398b4f301.png)

However, my Terminal is definitely capable of printing in 256 colors, as demonstrated by running this script:

```
$ ( x=`tput op` y=`printf %$((${COLUMNS}-6))s`;for i in {0..256};do o=00$i;echo -e ${o:${#o}-3:3} `tput setaf $i;tput setab $i`${y// /=}$x;done; )
```

![Screen Shot 2020-07-06 at 5 23 32 PM](https://user-images.githubusercontent.com/31248790/86657400-d27ee380-bfad-11ea-94eb-d9ed9dd34ed8.png)

##### Affected versions (if applicable)
- [ ] `master` branch (specify commit)
- [x] Latest stable version from `pypi`
- [ ] Other (specify source)

##### Steps to reproduce (if applicable)
Run 

##### Expected/actual outcome
Palette Test:
http://urwid.org/_images/palette_test2.png

High Color Mode Example:
http://urwid.org/_images/highcolors1.png

**Comment by tonycpsu** [comment]
Must be something with your terminal, as the output looks like the screenshot for me.

**Comment by dimays** [comment]
Any settings you'd recommend I check? Or a way to emulate a different terminal to test it?

I figured it couldn't be an issue with the Terminal, since it displays 256 colors just fine when not going through Urwid.

**Comment by tonycpsu** [comment]
Oh, you edited your post to include the fact that you're running Terminal.app. This was a known issue with the 2.1.0 release.  Fixed in #404, so wait for the 2.1.1 release or install from the master branch.

**Comment by dimays** [comment]
Nice, thanks!
